samples/openvx/openvx_interoperability.cpp

```diff
@@ -0,0 +1,324 @@
+#include <iostream>
+#include <stdexcept>
+
+//OpenVX includes
+#include <VX/vx.h>
+#include <VX/vx_lib_debug.h>
+#include <VX/vx_helper.h>
+#include <VX/vx_lib_extras.h>
+
+//for vx-Access/Commit-ImagePatch
+#include <VX/vx_compatibility.h>
```

Comment:
Since we didn't settled on what OpenVX version we should support I used 1.0 functions.
Now we've decided to support OpenVX from v1.1 only.